src/style.css

```diff
@@ -2,2 +1,76 @@
+body,
+header,
+a,
+ul {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  /* stylelint-disable-next-line font-family-name-quotes */
+  font-family: 'Roboto', 'Arial', sans-serif;
```

Comment:
Great job on using fallback fonts - alternative font-family in case the main one doesn't work.